mkdocs/docs/HPC/running_batch_jobs.md

```diff
@@ -424,24 +421,23 @@ To see how a module would change the environment, you can use the
 `module show` command:
 
 ```
-$ module show Python/2.7.12-intel-2016b
-whatis("Description: Python is a programming language that lets youwork more quickly and integrate your systems more effectively. - Homepage: http://python.org/ ") 
-conflict("Python")
-load("intel/2016b") 
-load("bzip2/1.0.6-intel-2016b") 
+$ ml show Python/3.12.3-GCCcore-13.3.0
```

Comment:
missing `load` statements because modules were already loaded?

Please fix, and maybe use `Python-bundle-PyPI` module instead